BE/src/main/java/kr/codesquad/issuetracker/domain/Label.java

```diff
@@ -0,0 +1,31 @@
+package kr.codesquad.issuetracker.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+
+@Entity
+public class Label {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "label_id")
+	private Long id;
+
+	private String title;
+
+	private String description;
+
+	private String backgroundColor;
+
+	private String textColor;
+
+	@OneToMany(mappedBy = "label")
+	private List<IssueLabels> issueLabelsList = new ArrayList<>();
```

Comment:
아래 처럼 변수 명에서 타입을 제거하는게 유지관리 하기에 용이해 보입니다.

```suggestion
	private List<IssueLabel> IssueLabels = new ArrayList<>();
```

Reply:
아하 변경하도록 하겠습니다 감사합니다 :)